packages/preact/src/components/SearchInput.tsx

```diff
@@ -0,0 +1,29 @@
+import { isHTMLInput } from "@preact/dom/utils"
+import { ComponentChildren } from "preact"
+
+import { renderHeadless } from "../dom/renderHeadless"
+
+type Props = {
+  children: ComponentChildren
+  onSearchInput: (target: HTMLInputElement) => void
+}
+
+export function SearchInput({ children, onSearchInput }: Props) {
+  return renderHeadless({
+    children,
+    updateElement: vnode => {
```

Comment:
This signature of this method is set to return 
```ts
VNode<P> | null
```
but no indication in this method about handling null value 😕